ampd/Cargo.toml

```diff
@@ -70,3 +79,18 @@ random-string = "1.0.0"
 [build-dependencies]
 ethers = "2.0.8"
 tonic-build = "0.8.3"
+
+[patch.crates-io]
```

Comment:
We should probably put a NOTE or TODO here to remind ourselves to notify the Axelar team about this.
It's a rather hacky solution, but the only one so far.

[Issue on the solana project](https://github.com/solana-labs/solana/issues/26688)

Reply:
I think after syncing with `solana` this is already done.